routes/index.js

```diff
@@ -167,6 +167,7 @@ router.post('/captions/stop', postBodyParser, async function (req, res) {
       },
     });
     res.sendStatus(captionResponse.status);
+    captionsId = '';
```

Comment:
`captionsId` needs to be tied to a session and not in the global scope. You are creating a race condition when multiple sessions have captions enabled